### ISSUE TITLE
re-implement issue

### ISSUE THREAD

**Issue by superkevingit** [Phuong86/uav-deep-q#1]
Hi, Thanks for sharing!

I tried to re-implement the DQL_2agents.py, but I found the Q-Net could not converge. The loss becomes super large and the reward is super small. Did you come across this situation, how could I fix it?

BTW, I worked with Macbook and conda, my environment is shown below

`name: uav
channels:
  - defaults
dependencies:
  - _tflow_select=2.3.0=mkl
  - absl-py=0.13.0=py37hecd8cb5_0
  - astor=0.8.1=py37hecd8cb5_0
  - blas=1.0=mkl
  - c-ares=1.17.1=h9ed2024_0
  - ca-certificates=2021.10.26=hecd8cb5_2
  - cached-property=1.5.2=py_0
  - certifi=2021.10.8=py37hecd8cb5_0
  - dataclasses=0.8=pyh6d0b6a4_7
  - gast=0.2.2=py37_0
  - google-pasta=0.2.0=pyhd3eb1b0_0
  - grpcio=1.42.0=py37ha29bfda_0
  - h5py=3.6.0=py37h4a1dd59_0
  - hdf5=1.10.6=hdbbcd12_0
  - importlib-metadata=4.8.2=py37hecd8cb5_0
  - intel-openmp=2021.4.0=hecd8cb5_3538
  - keras-applications=1.0.8=py_1
  - keras-preprocessing=1.1.2=pyhd3eb1b0_0
  - libcxx=12.0.0=h2f01273_0
  - libffi=3.3=hb1e8313_2
  - libgfortran=3.0.1=h93005f0_2
  - libprotobuf=3.17.2=h2842e9f_1
  - markdown=3.3.4=py37hecd8cb5_0
  - mkl=2021.4.0=hecd8cb5_637
  - mkl-service=2.4.0=py37h9ed2024_0
  - mkl_fft=1.3.1=py37h4ab4a9b_0
  - mkl_random=1.2.2=py37hb2f4e1b_0
  - ncurses=6.3=hca72f7f_2
  - numpy=1.21.2=py37h4b4dc7a_0
  - numpy-base=1.21.2=py37he0bd621_0
  - openssl=1.1.1l=h9ed2024_0
  - opt_einsum=3.3.0=pyhd3eb1b0_1
  - pip=21.2.2=py37hecd8cb5_0
  - protobuf=3.17.2=py37h23ab428_0
  - python=3.7.11=h88f2d9e_0
  - readline=8.1=h9ed2024_0
  - scipy=1.7.1=py37h88652d9_2
  - setuptools=58.0.4=py37hecd8cb5_0
  - six=1.16.0=pyhd3eb1b0_0
  - sqlite=3.36.0=hce871da_0
  - tensorflow=1.14.0=mkl_py37h085be34_0
  - tensorflow-base=1.14.0=mkl_py37h5a24fda_0
  - tensorflow-estimator=1.14.0=py_0
  - termcolor=1.1.0=py37hecd8cb5_1
  - tk=8.6.11=h7bc2e8c_0
  - typing_extensions=3.10.0.2=pyh06a4308_0
  - webencodings=0.5.1=py37_1
  - werkzeug=0.16.1=py_0
  - wheel=0.37.0=pyhd3eb1b0_1
  - wrapt=1.13.3=py37hca72f7f_2
  - xz=5.2.5=h1de35cc_0
  - zipp=3.6.0=pyhd3eb1b0_0
  - zlib=1.2.11=h1de35cc_3
  - pip:
    - cachetools==4.2.4
    - charset-normalizer==2.0.9
    - cycler==0.11.0
    - fonttools==4.28.3
    - google-auth==1.35.0
    - google-auth-oauthlib==0.4.6
    - idna==3.3
    - kiwisolver==1.3.2
    - matplotlib==3.5.1
    - oauthlib==3.1.1
    - packaging==21.3
    - pillow==8.4.0
    - pyasn1==0.4.8
    - pyasn1-modules==0.2.8
    - pyparsing==3.0.6
    - python-dateutil==2.8.2
    - requests==2.26.0
    - requests-oauthlib==1.3.0
    - rsa==4.8
    - tensorboard==2.0.2
    - torch==1.10.0
    - tqdm==4.62.3
    - urllib3==1.26.7
`
